Makefile

```diff
@@ -103,6 +106,9 @@ signed_xpi: addon
 	./node_modules/.bin/web-ext sign --api-key=${AMO_USER} --api-secret=${AMO_SECRET} --source-dir addon/webextension/
 	mv web-ext-artifacts/*.xpi build/pageshot.xpi
 
+addon_locales: $(l10n_dest)
+	./bin/pontoon-to-webext.js --dest addon/webextension/_locales
```

Comment:
pontoon-to-webext should go in `bin/build-scripts/` since it isn't called directly, instead it's only called through the Makefile